docs/Tutorial_Consensus_Motif.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>We can probably remove <code>urllib</code> , <code>ssl</code> , <code>zipfile</code> , <code>requests</code> , and <code>loadmat</code> now since we are retrieving data from Zenodo (please see comments below)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Thanks!</p>